pom.xml

```diff
@@ -32,17 +32,11 @@
   <name>WildFly Swarm Booster - ConfigMap Mission</name>
 
   <properties>
-    <version.wildfly.swarm>2017.10.0</version.wildfly.swarm>
+    <version.wildfly.swarm>2018.3.3</version.wildfly.swarm>
     <version.resteasy>3.0.19.Final</version.resteasy>
-    <version.arquillian.cube>1.9.2</version.arquillian.cube>
-    <!-- Licenses - plugins -->
-    <version.wildfly.swarm.fraction.plugin>61</version.wildfly.swarm.fraction.plugin>
-    <version.xml-maven-plugin>1.0.1</version.xml-maven-plugin>
-    <version.license-maven-plugin>1.13</version.license-maven-plugin>
-    <version.wildfly-licenses-plugin>1.0.1</version.wildfly-licenses-plugin>
-    <!-- Licenses - JBoss repo URL used in custom-settings -->
-    <jboss.repo.url>https://repository.jboss.org/nexus/content/groups/public</jboss.repo.url>
-    <licenses.pom.file>pom.xml</licenses.pom.file>
```

Comment:
I wonder about these license changes. Do we have all this coming from the parent now?

Reply:
I removed them since no other community booster has them, thus concluded they are redundant. I guess these properties might be needed for the license regeneration ? Though I see no any refs to them in this pom.xml or in other boosters poms

Reply:
Good point, in the community booster, we don't really need to take care of generating licenses etc. I'll take a look in the `-redhat` boosters.